Answers.md

```diff
@@ -0,0 +1,11 @@
+1.	If you saw this HTML: <div class="box box1 box2 box3"></div> which class has the most specificity weight?
+The class selector.box3 has the most specificity weight. 
+2.	Describe the difference between display: block; and display: inline;.
+Display: block will cause the element to display on a new line and take up its full available width.  Display: inline will add elements to the same line, only taking up as much width as necessary. 
+3.	While using flexbox, what axis are you using when you use the property: align-items: center? 
+With ‘align’ you are using the vertical axis, and so your items would be centered within a column or up-and-down orientation. 
+4.	What is the difference between fixed layout, adaptive layout, fluid layout, and responsive layout?
+A fixed layout has elements that are ‘fixed’ and not responsive to different device screen sizes, different resolution settings/zooming in etc. If the user does not have a compatible screen width, they have to use scroll bars to navigate left and right. An adaptive layout adapts to different device screen sizes using media queries. A fluid layout has converted elements components that make them responsive to the changing widths of the screen and resolution settings.  
+5.	Why do we need to use the CSS property max-width on the outter most container in a responsive website? 
+A max-width on the outermost container keeps the resolution and orientation of the elements intact on larger screens. A width at this level in general provides a baseline for compartmentalizing your adaptive and fluid elements.  
```

Comment:
Max-width provides a constraint which is a word Josh has brought up during the lectures. Your definition is spot-on!